test-win-sshproxy/suite_test.go

```diff
@@ -1,4 +1,4 @@
-// +build windows
+//go:build windows
```

Comment:
In the other files, you kept `// +build windows`, any reason for removing it here? 
For now I would keep it, for consistency with the darwin/… files with have both tags.